src/background/index.ts

```diff
@@ -62,6 +70,7 @@ chrome.browserAction.onClicked.addListener(function() {
 //keyboard shortcut: triggers browser action
 
 chrome.commands.onCommand.addListener(function(command) {
+  window.analytics.track('Extension Shortcut')
```

Comment:
for some reason this is not getting fired. any idea why?

Reply:
are we sure this is anonymous function is firing in general? (the one you pass into the `onCommand.addListener(`

I would validate that assumption first then figure it out from there.

Reply:
Oh you're right! Realized i didn't need the function anyway because declaring it as "_execute_browser_action" in the manifest triggers the keyboard shortcut by itself

Reply:
Dope, good call